src/context.rs

```diff
@@ -1152,7 +1152,7 @@ mod test {
   }
 }
 
-const SUPERBLOCK_TO_PLANE_SHIFT: usize = MAX_SB_SIZE_LOG2;
+pub const SUPERBLOCK_TO_PLANE_SHIFT: usize = MAX_SB_SIZE_LOG2;
```

Comment:
`pub(crate)` just in case.

Reply:
All the other stuff there is `pub` rather than `pub(crate)`. I think someone needs to make a separate PR to change all unnecessary `pub` to `pub(crate)`.

Reply:
> All the other stuff there is `pub` rather than `pub(crate)`. I think someone needs to make a separate PR to change all unnecessary `pub` to `pub(crate)`.

For existing code, such a PR makes sense, but for newly added code we should start with limited scope from the outset.